section1/reflection.md

```diff
@@ -1,19 +1,42 @@
 ## Section 1 Reflection
 
-1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
-
-2. What are the data types you learned about in this section? In your own words, define each.
-
-3. How would you log the string `"Hello World!"` to the console?
-
-4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
-
-5. In your own words, what is a variable? How would you explain it to a 5 year old?
-
-6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
-
-7. In your own words, explain what concatenation is.
-
-8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
-
-9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?  
+I think I sometimes ask questions too soon but I think that's more a problem when I'm working independently than in groups where I feel comfortable waiting a second
+to see if anybody else has the same question and if they don't going "huh I wonder if there's some obvious thing I'm missing."  
+2. What are the data types you learned about in this section? In your own words, define each.  
+Strings- These are sets of characters that sometimes have some limitations but for the most part accept anything between '' or "" marks. Including numbers!  
+Numbers- Now numbers that aren't inside quotes are treated like, well, numbers. You can do mathematical operations on them and everything.  
+Booleans- These are logical statements like true or false. The computer can evaluate a statement to set something as true or you can simply set it as true or false immediately
+like menUsedToLookBetterInHats = true  
+
+3. How would you log the string `"Hello World!"` to the console?  
+I think this is a trick question? Because you use quotes to define something as a string using them within the string can confuse things so you have to be careful and use \ to mark special characters. So in order to log this exact string you'd use
+console.log('\`"Hello World!"\`').  
+If I'm overthinking this and the back ticks are just here to `format soemthing as code` then you could simply use single quotes to enclose the double quotes as in  
+console.log('"Hello World!"').
```

Comment:
☺️ You are awesome David! I think we were just looking for what you have on line 16, but nice job figuring out the other!